src/components/QuickId.vue

```diff
@@ -38,7 +38,6 @@ export default {
 <style scoped lang="scss">
 .quick-id {
   display: flex;
-  justify-content: space-around;
   align-items: center;
```

Comment:
`space-between` should be the one to do the trick. And for ` a` link set `flex: 0 0 auto` (if it does not work, just play around with the `flex: 0 0`